packages/style-engine/src/styles/color/gradient.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import type { Style, StyleOptions } from '../../types';
+import { generateRule, getSlugFromPreset } from '../utils';
+
+const text = {
+	name: 'gradient',
+	generate: ( style: Style, options: StyleOptions ) => {
+		return generateRule(
+			style,
+			[ 'color', 'gradient' ],
+			'gradient',
```

Comment:
The gradient block support currently stores the value in the `background` CSS property, so I think we'd need to update this line to:

```suggestion
			'background',
```